cimbar/encode/cimb_translator.py

```diff
@@ -121,40 +132,64 @@ def _check_color(self, c, d):
         #return (c[0] - d[0])**2 + (c[1] - d[1])**2 + (c[2] - d[2])**2
         return relative_color_diff(c, d)
 
-    def _scale_color(self, c, adjust, down):
+    def _scale_adjust(self, c, adjust, down):
         c = int((c - down) * adjust)
-        if c > (245 - down):
+        if c > 245:
             c = 255
+        if c < 0:
+            c = 0
         return c
 
-    def _correct_all_colors(self, r, g, b):
-        if self.ccm is not None:
-            r, g, b = self.ccm.dot(numpy.array([r, g, b]))
-        return r, g, b
-
-    def _best_color(self, r, g, b):
-        r, g, b = self._correct_all_colors(r, g, b)
-
-        # probably some scaling will be good.
+    def scale_color(self, r, g, b):
+        if self.disable_color_scaling:
+            return r, g, b
```

Comment:
This code base continues to be a victim of being used for prototyping things. This variable and logic can probably be deleted...
... unless it ends up being really useful, so we'd better keep it :no_mouth: